tests/test_fga.py

```diff
@@ -0,0 +1,465 @@
+        return {
+            "object": "list",
+            "data": [
+                {
+                    "type": "feature",
+                    "relations": {
+                        "member": {
+                            "inherit_if": "any_of",
+                            "rules": [
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "feature",
+                                    "with_relation": "member",
+                                },
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "pricing-tier",
+                                    "with_relation": "member",
+                                },
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "tenant",
+                                    "with_relation": "member",
+                                },
+                            ],
+                        }
+                    },
+                },
+                {
+                    "type": "permission",
+                    "relations": {
+                        "member": {
+                            "inherit_if": "any_of",
+                            "rules": [
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "permission",
+                                    "with_relation": "member",
+                                },
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "role",
+                                    "with_relation": "member",
+                                },
+                            ],
+                        }
+                    },
+                },
+                {
+                    "type": "pricing-tier",
+                    "relations": {
+                        "member": {
+                            "inherit_if": "any_of",
+                            "rules": [
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "pricing-tier",
+                                    "with_relation": "member",
+                                },
+                                {
+                                    "inherit_if": "member",
+                                    "of_type": "tenant",
+                                    "with_relation": "member",
+                                },
+                            ],
+                        }
+                    },
+                },
+            ],
+            "list_metadata": {"after": "{{ orderASCLimit3.list_metadata.after }}"},
```

Comment:
Is this the intended format of `list_metadata["after"]`?
```
{{ orderASCLimit3.list_metadata.after }}
```

Reply:
It is not - I copied these responses from some API tests and missed this part. Will update.